internal/service/handler.go

```diff
@@ -42,4 +50,8 @@ func (s *Handler) Register(r *mux.Router) {
 	r.Handle(APIPrefix+"/gathering_rules", gatheringRulesEndpoint(s.svc))
 	r.HandleFunc(APIPrefix+V1Prefix+"/openapi.json", serveOpenAPI)
 	r.Handle(APIPrefix+V1Prefix+"/gathering_rules", gatheringRulesEndpoint(s.svc))
+
+	ocpVersion := "TBD"
+	v2Path := fmt.Sprintf("%s%s/%s/gathering_rules", APIPrefix, V2Prefix, ocpVersion)
```

Comment:
I'd adapt this code to accept any `ocpVersion`

Reply:
then we can handle any exception inside the remoteConfigurationEndpoint

Reply:
Something like `%s%s/{ocpVersion}/gathering_rules` right? I can't then get `r.PathValue("ocpVersion")` for some reason